### PR TITLE
PDM 2184: script to aggregate financial transaction report

### PR DESCRIPTION
**Pull request by ekinesocotra**

### CODE REVIEW

**Comment by hylandm** [comment]
LGTM. Have you run this against a tenant? Did you verify its working?

**Comment by ekinesocotra** [comment]
Yup. It works... sample tenant: eakuiyibo-alteos-test1.co.sandbox.socotra.com

File: reporting/create_aggregate_FT_report.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    parser.add_argument('-s', '--startdate', required=True)
+    parser.add_argument('-e', '--enddate', required=True)
+    parser.add_argument('-o', '--outputfile', required=True)
+
+    return parser.parse_args()
+
+
+def get_report_timestamps(startdate, enddate):
+    start_timestamp = dates.date_to_millis(startdate, 'Europe/Berlin', "%Y-%m-%d")
+    end_timestamp = dates.date_to_millis(enddate, 'Europe/Berlin', "%Y-%m-%d")
```
Comment (dineshshenoy) [inline reporting/create_aggregate_FT_report.py]: Can we change these to "Americas/Los Angeles"?

Comment (ekinesocotra) [reply reporting/create_aggregate_FT_report.py]: why?

**Pull request merged**